app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -273,12 +277,24 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
 
   var getStorageManagerFormData = function(data) {
     vm.cloudVolumeModel.emstype = data.type;
-    vm.cloudTenantChoices = data.cloud_tenants;
-    vm.availabilityZoneChoices = data.volume_availability_zones;
-    vm.baseSnapshotChoices = data.cloud_volume_snapshots;
+    vm.storageResources = data.storage_resources;
+    vm.storageServices = data.storage_services;
+    if (data.parent_manager){
```

Comment:
Missing space after `)`

Reply:
@skateman , shouldn't this be handled by a linter?

Reply:
It should be, we have an eslint config that can be integrated into your editor. Unfortunately, miq-bot is not capable of running eslint yet, but there's some ongoing work to have it there too.